pkg/test/framework/components/istio/operator.go

```diff
@@ -253,6 +290,48 @@ func deployControlPlane(c *operatorComponent, cfg Config, cluster kube.Cluster,
 	return nil
 }
 
+// replaceIstiodRemoteService deletes and recreates the istiod-remote Service and Endpoint with two key differences:
+// - The Service does not user `ClusterIP: None` so we can map 15012 to the configured port as `targetPort`
+// - The Endpoint uses the given port rather than the default 15012
+func replaceIstiodRemoteService(cluster kube.Cluster, address net.TCPAddr) error {
```

Comment:
Just seeing if this works in CI. Got it working locally with KinD. I will probably clean this up so:
- we don't have to delete/recreate 
- we can keep the Istio labels